package/gluon-alfred/files/lib/gluon/cron/alfred

```diff
@@ -1 +1 @@
-* * * * * /lib/gluon/announce/collect.lua nodeinfo | gzip | alfred -s 158; /lib/gluon/announce/collect.lua statistics | gzip | alfred -s 159; /lib/gluon/announce/collect.lua neighbours | gzip | alfred -s 160
+* * * * * gluon-neighbour-info -d ::1 -p 1001 -t 0 -r nodeinfo | gzip | alfred -s 158; gluon-neighbour-info -d ::1 -p 1001 -t 0 -r statistics | gzip | alfred -s 159; gluon-neighbour-info -d ::1 -p 1001 -t 0 -r neighbours | gzip | alfred -s 160
```

Comment:
What will gluon-neighbour-info do with `-t 0` when the request packet gets lost? While that shouldn't happen normally, won't that make the command block indefinitely when
* respondd is not running or
* the request or answer is dropped because of full queues?